scripts/items/loot/rf_geist_tear_item.nut

```diff
@@ -0,0 +1,77 @@
+this.rf_geist_tear_item <- this.inherit("scripts/items/item", {
+	m = {},
+	function create()
+	{
+		this.item.create();
+		this.m.ID = "misc.rf_geist_tear";
+		this.m.Name = "Geist Tear";
+		this.m.Description = "When a geist leaves this plane, sometimes a gem is left behind. Wise men believe it is the ethereal essence of the spirit made manifest.";
+		this.m.Icon = "loot/rf_geist_tear.png";
+		this.m.SlotType = this.Const.ItemSlot.None;
```

Comment:
lots of `this.Const` uses etc. in this file. 
Auto replacing them just within this file is quickly done